.github/workflows/cd.yaml

```diff
@@ -0,0 +1,58 @@
+name: Deploy Static Website to S3
+
+on:
+  pull_request:
+    branches: '**'
+
+# These permissions are needed to interact with GitHub's OIDC Token endpoint.
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy:
+    strategy: # FIXME
+      matrix:
+        environment: [ dev ]
+    name: Deploy website to S3
+    runs-on: ubuntu-latest
+    environment: ${{ matrix.environment }}
+
+    steps:
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+
+      - name: Setup node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
+        with:
+          node-version-file: '.node-version'
+          cache: 'npm'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Compile packages
+        run: npm run compile
+
+      - name: Build static website
+        env:
+          GITBOOK_API_KEY: ${{ secrets.GITBOOK_API_KEY }}
+        run: npm run build -w nextjs-website
+
+      - name: Export static website
+        env:
+          GITBOOK_API_KEY: ${{ secrets.GITBOOK_API_KEY }}
```

Comment:
I'm not sure but I think that the `export` command doesn't need the `GITBOOK_API_KEY`

Reply:
You are right, the API key is required during the build phase, where we fetching content from GitBook. I've removed from this step.